src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -86,13 +92,12 @@ describe('<PillButton> component', () => {
             expect(getButtonByName(upArrowLabel)).toHaveClass(
                 'PillButton__button PillButton__button--isOpen'
             );
-            expect(getButtonByName('Pill name up arrow')).not.toHaveClass(
-                'PillButton__button--isOpen'
-            );
+            expect(screen.queryByText(upArrowLabel)).not.toBeInTheDocument();
+            expect(screen.queryByText(clearLabel)).not.toBeInTheDocument();
         });
 
         it('should have arrow up label', () => {
-            expect(screen.getByRole('img')).toBeInTheDocument();
+            expect(screen.getByRole(imgRole)).toBeInTheDocument();
             expect(getButtonByName(upArrowLabel)).toBeInTheDocument();
         });
     });
```

Comment:
In the below test `should render correctly` can you add a check for which arrow is showing?

Reply:
:+1:

Reply:
this is not yet addressed; note that line is not visible in the diff in the Github UI.